react/react1/week3/to-do-app/src/Main.jsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from "react";
+
+  var now = new Date();
+    var day = ("0" + now.getDate()).slice(-2);
+    var month = ("0" + (now.getMonth() + 1)).slice(-2);
+    var today = now.getFullYear() + "-" + month + "-" + day;
+
+export default function Main() {
+    const [listItem, setListItem] = useState([]);
+    const [input, setInput] = useState('');
+    const [update, setUpdate] = useState('')
+    const [deadline, setDeadline] = useState(today)
+
+    const DeleteList = (id) => {
```

Comment:
You are deleting an item from the list, I suggest you change it to Item. 
```suggestion
    const DeleteItem = (id) => {
```